antiplag/antiplag/documents.py

```diff
@@ -0,0 +1,21 @@
+from django_elasticsearch_dsl import Document
+from django_elasticsearch_dsl.registries import registry
+from .models import Document as DocumentModel
+
+
+@registry.register_document
+class DocumentModelDocument(Document):
+    class Index:
+        # TODO: implement
+
+        # Name of the Elasticsearch index
```

Comment:
Are you planning on implementing this as a part of this task? Or we can merge it and implement later?

Reply:
We agreed in Slack on implementing it in other task.